### PR TITLE
Integrate Rafter in Kyma

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:

Integrate Rafter in Kyma:
  - enable installation of Rafter
  - disable installation on of `asset-store` and `cms`
  - update image of `cbs` - create `rafter` domain in `console-backend-service`
  - enable integration tests of `rafter` domain in `console-backend-service-tests`
  - update `helm-broker` image - integration with rafter
  - update relevant rbacs
  - change old resources from AssetStore/HCMS to new one in Rafter - create migration job
  - update image of `content-ui`, `service-catalog-ui` with new `rafter` queries in `cbs`
  - update image of `ui-tests` - update tests after introduce new queries
  - write and enable rafter integration tests
  - write tests for Rafter in e2e-upgrade tests (with checking behaviour of migration job)
  - write tests for Rafter in e2e-backup tests and removing old from AssetStore and HCMS
  - migrate compass-runtime to Rafter

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/6239

### CODE REVIEW

**Comment by magicmatatjahu** [comment]
This PR is temporary blocked by https://github.com/kyma-project/kyma/pull/6485 - needed new ENVS in cbs-tests.

File: tests/end-to-end/backup-restore-test/deploy/chart/backup-test/templates/test/test.yaml [hunk 1]
```diff
@@ -38,7 +38,8 @@ spec:
           sleep 4
 
           exit $exit_code
-        image: "{{ .Values.containerRegistry.path }}/{{ .Values.image.dir }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
+        # image: "{{ .Values.containerRegistry.path }}/{{ .Values.image.dir }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
+        image: "eu.gcr.io/kyma-project/pr/tests/backup-test:PR-6538"
```
Comment (suleymanakbas91) [inline tests/end-to-end/backup-restore-test/deploy/chart/backup-test/templates/test/test.yaml]: don't forget to revert this :)

**Comment by magicmatatjahu** [comment]
test /pre-master-kyma-gke-upgrade

**Pull request merged**